download_data.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# czi-cancer-data
+# 2018 Gregory Way
+
+# Script to download gene expression and covariate data from 4 single cell gene
+# expression analyses. The script will fetch, decompress, and rename each data set
+# listed below
+
+mkdir --parents data/raw
```

Comment:
Should this be followed by `cd data && cd raw`?

Reply:
i process files in this script too - it moves the data to appropriate directory

Reply:
Oh yea, my bad